src/parsec.rs

```diff
@@ -183,15 +219,23 @@ impl ParsecMap {
             .get_mut(&version)
             .ok_or(CreateGossipError::MissingVersion)?
             .create_gossip(target)?;
+
+        if version == self.last_version() {
+            self.gossip_count += 1;
```

Comment:
That's right. I don't think this is an issue because the vast majority of gossip traffic comes from the latest instance, not the old ones. But should not be too hard to change in case we want to.